app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx

```diff
@@ -14,36 +14,51 @@ const iconMap = {
 
 type FixedConfirmationToastProps = {
   className?: string,
+  closeable?: boolean,
   data?: string,
   id?: string,
   status?: "success" | "error" | "neutral" | "tip",
   text: string,
 }
 
 const FixedConfirmationToast = (props: FixedConfirmationToastProps) => {
-  const { className, status = 'neutral', text } = props
+  const [showToast, toggleToast] = useState(true)
+  const { className, closeable = false, status = 'neutral', text } = props
   const css = classnames(
     `pb_fixed_confirmation_toast_kit_${status}`,
     globalProps(props),
-    className
+    className,
+    closeable && 'remove_toast'
   )
   const icon = iconMap[status]
 
   return (
-    <div className={css}>
-      <If condition={icon}>
-        <Icon
-            className="pb_icon"
-            fixed_width
```

Comment:
Fixed. This is a rails style prop. : /

Reply:
Is this related to the failed milano build? Just checking. @coleerikson

Reply:
I believe it's unrelated. Testing and rerunning Milano.